packages/terra-date-time-picker/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ChangeLog
 =========
+### Added
+* Added new `timeZone` prop which takes a valid timezone string. This enables date-time-picker set to a particular timezone. The default timezone  is set to the local/browser timezone.
```

Comment:
This should be moved to unreleased section.

Reply:
Changed here b5f07bc5f04b127f3d75bc46a3651a2e1091b165